### PR TITLE
First version of seneka_dgps

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: seneka_windsensor/manifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<package>
+  <description brief="seneka_dgps">
+
+     This package published a laser scan message out of a Sick S300 laser scanner.
+
+  </description>
+  <author>Ciby Mathew</author>
+  <license>LGPL</license>
+  <review status="unreviewed" notes=""/>
+  <url>http://ros.org/wiki/cob_sick_s300</url>
```
Comment (floweisshardt) [inline seneka_windsensor/manifest.xml]: sure this is the correct url?

File: seneka_dgps/manifest.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<package>
+  <description brief="seneka_dgps">
+
+     This package published a laser scan message out of a Sick S300 laser scanner.
+
+  </description>
+  <author>Ciby Mathew</author>
+  <license>LGPL</license>
+  <review status="unreviewed" notes=""/>
+  <url>http://ros.org/wiki/cob_sick_s300</url>
```
Comment (floweisshardt) [inline seneka_dgps/manifest.xml]: sure this is the correct url?

Comment (ghost) [inline seneka_dgps/manifest.xml]: Hello Florian,

Yes, this is the final working one. 

Best regards

Ciby

Comment (mgruhler) [inline seneka_dgps/manifest.xml]: Hey Ciby,

check the URL in the manifest.xml of your packages. It is definitely not the right one.

Best,
Matthias

---

Dipl.-Ing. Matthias GRUHLER, Fraunhofer IPA
Abteilung 323/Roboter- und Assistenzsysteme
Nobelstraße 12, 70569 Stuttgart
Telefon: +49 711 970-1315, Telefax: +49 711 970-1008
Matthias.Gruhler@ipa.fraunhofer.de  http://www.ipa.fhg.de/

-----Original Message-----

**Comment by fmirus** [comment]
This version seems to be outdated, so I closed this Pull Request.

**Pull request closed without merging**